docs/ui/components/EASHostingShoutoutBanner.tsx

```diff
@@ -74,7 +75,7 @@ export function EASHostingShoutoutBanner() {
           <Cloud01Icon className="icon-lg relative z-10 text-palette-white" />
         </div>
         <div className="relative grid grid-cols-1">
-          <p className="font-medium text-success heading-base">EAS Hosting</p>
+          <HEADLINE className="text-success">EAS Hosting</HEADLINE>
```

Comment:
```suggestion
          <p className="font-medium text-success text-base">EAS Hosting</p>
```
`HEADLINE` is a legacy text component, we should avoid using that. The problem generates the `heading-base` definition in styleguide which allows to small values. If you want to fix this ASAP, without styleguide fix, we can just switch to `text-base` class instead for now.